### PR TITLE
Afsu

### PR DESCRIPTION
**Pull request by Afshara**
Bug #3, cell dimension

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -80,6 +80,760 @@ function makeMove(element) {
...
+
+        // TESTING PURPOSES
+        // var cols = document.getElementsByClassName('ChessBoard')[0].getElementsByTagName('td');
+        // for(let i = 0; i < cols.length; i++){
+        //     // console.log(cols[i].childNodes[0].lastElementChild.id);
+        //     document.getElementsByTagName("td")[i].innerHTML += cols[i].childNodes[0].lastElementChild.id
+        // }
+    }
+}
+
```
Comment (engxx) [inline main.js]: Didn't we create separate this into different files? Why did you put it back to the main file?

**Comment by Afshara** [comment]
Fixed the bug in the correct version of master

File: main.css [hunk 2]
```diff
@@ -25,6 +25,13 @@ table, th , td{
 th, td{
     text-align: left;
 }
+
 .BoardBlock{
     height: 80px;
+}
+
+.blockDimension{
```
Comment (engxx) [inline main.css]: Is there a reason for camelCase? we should stay consistent cause the other ones are Pascal

Comment (Afshara) [reply main.css]: no reason, it's okay to change it back

**Pull request closed without merging**